src/applications/bmqstoragetool/m_bmqstoragetool_filemanager.h

```diff
@@ -191,7 +191,7 @@ inline FileManagerImpl::FileHandler<ITER>::FileHandler(
 }
 
 template <typename ITER>
-inline FileManagerImpl::FileHandler<ITER>::~FileHandler()
+inline FileManagerImpl::FileHandler<ITER>::FileHandler::~FileHandler()
```

Comment:
I think this syntax doesn't compile on C++03, I believe I've been burned by this before.

Reply:
I'll give this a shot in C++03 then; if it doesn't compile, we'll have to turn that warning off.

I see in the git log that we also tried:
```
inline FileManagerImpl::FileHandler<ITER>::~FileHandler<ITER>()
```
which also doesn't compile in C++03.

Reply:
Can we solve this issue completely by placing the nested class declaration to the outer scope?